### ISSUE TITLE
Migration Settings page goes blank, highlights wrong menu selection in Vertical Navigation

### ISSUE THREAD

**Issue by kedark3** [ManageIQ/manageiq-v2v#802]
https://bugzilla.redhat.com/show_bug.cgi?id=1651403

**Comment by vconzola** [comment]
@mturley @michaelkro @mzazrivec Think Milan's PR will fix at least the first part of this, right? What about the second part?

**Comment by mzazrivec** [comment]
That 5.10.0.24 build doesn't include my changes from https://github.com/ManageIQ/manageiq-v2v/pull/769 Correct?

**Comment by mturley** [comment]
I can confirm locally that both parts of this issue were resolved by #769. The second part (input field going blank) might have been something related to switching views with ConnectedRouter and re-rendering the settings page without a change to the stored settings in Redux. It does reliably work on a fresh page reload, though, and after #769 we do a full page reload on every nav change, so that fixes the input going blank as a result.

@mzazrivec, after we switch back to ConnectedRouter, the input going blank here might become an issue again.

**Comment by kedark3** [comment]
![image](https://user-images.githubusercontent.com/8180469/48793060-cdc7c780-ecc3-11e8-8e13-50c387b02192.png)
I think we still have issue. tested in the build master.20181116223021_0f85990

**Comment by vconzola** [comment]
Nav being highlighted wrong is fixed by Milan's first PR. The second part, we need to validate this is still fixed once Milan's new PR gets merged. @mturley to validate.

**Comment by mturley** [comment]
This bug is not present in `hammer`, it was hidden by #769 as indicated above. Milan's new PR #822 re-exposes this bug, but we are not merging that to `hammer`. I have a new PR #839 to fix the bug in `master`.
